tests/lax_numpy_test.py

```diff
@@ -5857,6 +5865,84 @@ def testUfuncInputTypes(self, name, arg_dtypes):
       # that jnp returns float32. e.g. np.cos(np.uint8(0))
       self._CheckAgainstNumpy(np_op, jnp_op, args_maker, check_dtypes=False, tol=1E-2)
 
+  @parameterized.named_parameters(
+    dict(testcase_name=f"_{name}_{dtype.__name__}_{region}", name=name, dtype=dtype, region=region)
+    for name, dtype, region in itertools.product(
+        _complex_numpy_ufuncs,
+        jtu.dtypes.supported([np.complex64, np.complex128]),
+        ['q1', 'q2', 'q3', 'q4',
+         'pos', 'neg', 'posj', 'negj',
+         'ninf', 'pinf', 'ninfj', 'pinfj',
+         'zero']
```

Comment:
It looks like this generates ~700 test cases, then skips many of them in the test body. In the past we had to spend some time optimizing tests because this kind of overhead was becoming too expensive. Can you check how much runtime this adds to `lax_numpy_test`?

Also, would this possibly fit better in `lax_test`, since it is evaluating the numerics of `lax` implementations rather than anything particular to the `lax_numpy` wrappers?

Reply:
Running `JAX_PLATFORM_NAME='cuda' JAX_ENABLE_X64=1 pytest -sv tests/lax_numpy_test.py -k testUfuncOnComplexPlane` takes 7 seconds here.

When all the skips will be resolved, testing specific regions can be refactored to the whole complex plane and by that reducing the number of generated tests 13 times, that is, to 52 test cases.

The tests can be certainly moved to lax_test.

Reply:
7 seconds is probably fine, so long as we plan to resolve the skips in the near future.

Reply:
@hawkinsp might have opinions as the person who put the most effort into optimizing tests.

Reply:
Actually, the number of tests could be reduced already now be refactoring the specific regions to sets of regions: one set that captures matching cases, and another set of regions that will be skipped due to inaccuracies.

Reply:
@jakevdp , I have applied the following changes:
- moved the tests to lax_test.py
- reduced the total number of tests to 104. When all inaccuracy issues are fixed, the number can be reduced to 52: a test case per function per complex dtype.

Reply:
Thanks!